_includes/schemas/shipments_response.json

```diff
@@ -87,23 +83,40 @@
     "reference_number": {
       "type": "string",
       "description": "a reference number (max. 30 characters) that you want this shipment to be identified with. You can use this afterwards to easier find the shipment in the shipcloud.io backoffice"
+    },
+    "service": {
+      "type": "string",
+      "description": "the service that was used for creating this shipment"
+    },
+    "shipper_notification_email": {
+      "type": "string",
+      "description": "email address of the seller who should be notified of a change of shipment status by shipcloud"
```

Comment:
I feel like we should use the word "shipper" instead of "seller". What do you think?